environment.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
  - numpy=1.17.5
  - matplotlib=3.3.2
  - conda-forge::pyqtgraph=0.11.0
+ - conda-forge::pyvisa=1.10.1
```

Comment:
Did we want to revert to conda or should we keep it as pip?

Reply:
pyvisa 1.11.x is not avialble yet at conda-forge, so in #2363 we decided to just use pip for pyvisa.

Reply:
but if we support 1.10.1 again we could revert to using conda forge ?

Reply:
we could, but then folks using the environment.yml won't be using the latest greatest pyvisa :)

Reply:
okay let's keep as pip for 1.11 then.